cmd/vc-rest/startcmd/start.go

```diff
@@ -388,6 +389,21 @@ func buildEchoHandler(
 		return nil, err
 	}
 
+	vcCrypto := crypto.New(conf.VDR, documentLoader)
+
+	vcStatusStore := vcstatusstore.NewStore(mongodbClient)
+
+	cslManager, err := cslmanager.New(
```

Comment:
I think you need to move this to the credentialstatus component so that it may be overridden.

Reply:
@bstasyszyn it's look correct

Reply:
> @bstasyszyn it's look correct

The CSL manager should be overridable in the credentialstatus component, so it should be moved to credentialstatus.

Reply:
Done